app/mailers/contact_mailer.rb

```diff
@@ -1,17 +1,17 @@
 class ContactMailer < ActionMailer::Base
-  default from: "The Odin Project <admin@theodinproject.com>"
-  FEEDBACK_EMAIL = "feedback@theodinproject.com"
-  CONTACT_EMAIL = "contact@theodinproject.com"
-  ODIN_URL = "http://www.theodinproject.com"
+  default from: 'The Odin Project <admin@theodinproject.com>'
 
-
-  def suggestion_email(suggestion_text, url="", user_email="")
+  def suggestion_email(suggestion_text, url='', user_email='')
```

Comment:
I just had a look at `StaticPagesController`. Looks like we'll always always have a `url` and `user_email`(not always an email) too. Please checkout `suggestion` and `user_identifier` methods in `static_pages_controller.rb`